recipes/openturns/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "1.7" %}
+{% set variant = "openblas" %}
+
+package:
+  name: openturns
+  version: {{ version }}
+
+source:
+  fn: openturns-{{ version }}.tar.gz
+  url: https://github.com/openturns/openturns/archive/v{{ version }}.tar.gz
+  sha1: 9535120721b20a45460538f3ed8fa0f0fec1d797
+# see https://github.com/openturns/openturns/pull/206:
+  patches:
+    - openturns-config-path.patch
```

Comment:
Seems like a very small piece of that PR. Could you please clarify why this piece?